ec2-spot-deep-learning-training/ec2-spot-deep-learning.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash -x
```

Comment:
add yourself as an author. if possible, please add a few comments of what is this file.